Sources/Testing/Running/Runner.swift

```diff
@@ -188,30 +184,30 @@ extension Runner {
     defer {
       if let step = stepGraph.value {
         if shouldSendTestEnded {
-          Event.post(.testEnded, for: (step.test, nil), configuration: configuration)
+          Event.post(.testEnded, for: (step.test, nil), configuration: _configuration)
```

Comment:
If these events are all pulling the configuration from the `current` task-local, there's no need to specify it as an argument since that's the default.

Reply:
I'm hesitant to change all these `Event.post()` calls without a better understanding of what perf cost it might have. But I think I can safely capture the current configuration once in a local variable for the functions which need it, and that'll further reduce diffs needed here

Reply:
You're already paying the cost of the task-local here anyway.